src/configure/utilities/webAppNodeVersionConverter.ts

```diff
@@ -0,0 +1,36 @@
+import { JSONPath } from 'jsonpath-plus';
+import * as vscode from 'vscode';
+import { ArmRestClient } from '../clients/azure/armRestClient';
+import { AzureSession } from "../model/models";
+import { Messages } from '../resources/messages';
+
+export async function webAppRuntimeNodeVersionConverter(nodeVersion: string, armUri: string, azureSession: AzureSession): Promise<string> {
+    if (nodeVersion.indexOf('|') >= 0){
+        nodeVersion = nodeVersion.split('|')[1];
```

Comment:
If the first part is not node, then we should always convert to latest lts

Reply:
If first part is not node, can the nodeVersion be any standard version? 
I am returning nodeVersion as it is if that is the case.

Reply:
If first part is not node, then it will lostly be not valid node version . Hence I thought we can do lts

Reply:
okk done